### PR TITLE
OLS-1114 - Adding 4.15, 4.17 and changing the name of the 4.16 pipeline

### PR DESCRIPTION
**Pull request by JoaoFula**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ x] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ x] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by tisnik — COMMENTED** [review]
it looks ok

File: .tekton/integration-tests/pipelines/lightspeed-service-integration-test-pipeline-4.17.yaml [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+                value: "tests/test_results"
+              - name: SUITE_ID
+                value: "nosuite"
+              - name: KONFLUX_OLS_SERVICE_IMAGE
+                value: $(params.serviceimage)
+            image: registry.access.redhat.com/ubi9/ubi-minimal
+            script: |
+              echo "---------------------------------------------"
+              microdnf -y install git make python3.11 python3.11-devel python3.11-pip shadow-utils tar
+              curl -L -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.16/openshift-client-linux-amd64-rhel9.tar.gz \
```
Comment (raptorsun) [inline .tekton/integration-tests/pipelines/lightspeed-service-integration-test-pipeline-4.17.yaml:271]: should we use 4.17 client for 4.17 cluster?

File: .tekton/integration-tests/pipelines/lightspeed-service-integration-test-pipeline-4.15.yaml [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+                value: "tests/test_results"
+              - name: SUITE_ID
+                value: "nosuite"
+              - name: KONFLUX_OLS_SERVICE_IMAGE
+                value: $(params.serviceimage)
+            image: registry.access.redhat.com/ubi9/ubi-minimal
+            script: |
+              echo "---------------------------------------------"
+              microdnf -y install git make python3.11 python3.11-devel python3.11-pip shadow-utils tar
+              curl -L -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.16/openshift-client-linux-amd64-rhel9.tar.gz \
```
Comment (raptorsun) [inline .tekton/integration-tests/pipelines/lightspeed-service-integration-test-pipeline-4.15.yaml:271]: 4.16 client should be backward compatible so i'm not worried about this :P

**Comment by raptorsun** [comment]
/override "Red Hat Konflux / ols-e2e-tests-416 / test-bundle"
/override "Red Hat Konflux / ols-enterprise-contract / bundle"
/overide "Red Hat Konflux / ols-enterprise-contract / test-bundle"
/override "Red Hat Konflux / ols-enterprise-contract / pr group konflux-adding-test-4.15-4.17"

File: .tekton/integration-tests/pipelines/lightspeed-service-integration-test-pipeline-4.17.yaml (hunk 1 above)
Comment (JoaoFula) [reply .tekton/integration-tests/pipelines/lightspeed-service-integration-test-pipeline-4.17.yaml:271]: :o yessss!!! Good catch

**Comment by raptorsun** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / bundle"
/overide "Red Hat Konflux / ols-enterprise-contract / test-bundle"
/override "Red Hat Konflux / ols-enterprise-contract / pr group konflux-adding-test-4.15-4.17"

**Comment by raptorsun** [comment]
/override "Red Hat Konflux / ols-operator-e2e-tests-415 / test-bundle"

**Comment by raptorsun** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / bundle"
/override "Red Hat Konflux / ols-enterprise-contract / pr group konflux-adding-test-4.15-4.17"
/override "Red Hat Konflux / ols-enterprise-contract / test-bundle"
/override "Red Hat Konflux / ols-operator-e2e-tests-415 / test-bundle"

**Pull request merged**